backend/src/v5/models/revisions.js

```diff
@@ -45,4 +67,25 @@ Revisions.getRevisionCount = (teamspace, model) => {
 	return db.count(teamspace, collectionName(model), query);
 };
 
+Revisions.getRevisions = (teamspace, model, showVoid, projection = {}) => {
+	const query = { ...excludeIncomplete };
+
+	if (!showVoid) {
+		query.void = { $ne: true };
```

Comment:
use the constants up top
```suggestion
		query.void = excludeVoids.void
```